engine/core/src/main/java/org/datacleaner/util/sort/SortMergeWriter.java

```diff
@@ -45,8 +46,6 @@
  * Sorter, deduplicator and writer that uses temporary files as storage to
  * support high volume sorted data.
  * 
- * Note: This class is NOT thread-safe.
```

Comment:
Now it is :)

(what happened here? It doesn't seem related at all, was it just something the new UI uncovered?)